static/src/stylesheets/module/content/_article-minute.scss

```diff
@@ -571,8 +571,12 @@ $minute-layout-margin-nudge: gs-span(1);
                     width: 100%;
                 }
 
-                .gu-image {
+                .element--inline__outer {
```

Comment:
Do we add `element`to state?
Seems like `element--inline-outer` would make more BEM sense?